about.html

```diff
@@ -0,0 +1,207 @@
+    <title>About - Aruba Ruby Conference 2023</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="./assets/stylesheets/app.css" />
+    <link rel="stylesheet" href="./assets/stylesheets/about.css" />
+  </head>
+  <body>
+    <header>
+      <div id="nav-banner">
+        <a
+          href="https://www.facebook.com/"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <i class="fa-brands fa-facebook-f"></i>
+        </a>
+        <a
+          href="https://www.twitter.com"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <i class="fa-brands fa-twitter"></i>
+        </a>
+        <a href="#">
+          <span>English</span>
+        </a>
+        <a href="#">
+          <span>My Page</span>
+        </a>
+        <a href="#">
+          <span>Logout</span>
+        </a>
+      </div>
+      <nav id="navbar">
+        <i id="menu-btn" class="fa-solid fa-bars"></i>
+        <a href="./" id="logo">
+          <img src="./assets/images/logo.png" alt="logo" />
+        </a>
+        <ul>
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li id="cc_campaign"><a href="#">RoR Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section>
+      <div id="mobile-menu" class="d-none flex-column">
+        <header>
+          <div id="close-menu-btn">
+            <i class="fa-solid fa-x"></i>
+          </div>
+        </header>
+        <ul id="mobile-links">
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li id="cc_campaign"><a href="#">RoR Campaign</a></li>
+        </ul>
+      </div>
+    </section>
+
+    <section id="about-hero">
+      <header id="about-hero-header">
+        <h3 class="mobile-only">"Hello World!"</h3>
+        <h1>Aruba Ruby</h1>
+        <h1>Global summit 2023</h1>
```

Comment:
- [ ] To follow html best practice, there should be only one `h1` tag in each page of any website. You can use `<br />` to divide your main title or another approach. Kindly remove one `h1` tag and find a way to make it look the same without repeating the same `h1` tag twice.